docs/website/docs/guides/ml-frameworks/pytorch.md

```diff
@@ -322,6 +322,132 @@ their values independently at runtime.
         self.value = new_value
     ```
 
+#### :octicons-code-16: Separating parameters during compilation
+
+In practical scenarios, it is usually unfeasible to embed model parameters
+in the IR.
+
+`aot.externalize_module_parameters()` allows for the separation of
+the parameters from the IR, but still encodes a symbolic relationship between
+between parameters and the IR, so they can be loaded at runtime.
+
+We define a `torch.nn.Module` that will be compiled, and save
+random tensors to be loaded as parameters in a safetensors file.
+
+```python
```

Comment:
To keep the overall page organized, I would either put this entire example into a single code block:
```python
# Define a torch.nn.Module with some parameters.
import torch
class LinearModule(torch.nn.Module):
    ...

# Save the parameters to a .safetensors file.
...
save_file(params, "params.safetensors")

# Externalize the model parameters.
aot.externalize_module_parameters(linear_module)
...
```

or break out each step into an ordered or unordered list:

````
1. We define a `torch.nn.Module` that will be compiled, and save
random tensors to be loaded as parameters in a safetensors file.

    ```python
    import torch
    ...
````

Here's how the page would look with a list:
![image](https://github.com/user-attachments/assets/ca70bbb2-80d5-419a-a75f-06f0179f53a2)